src/routes/ChatFinished/container.js

```diff
@@ -43,8 +44,9 @@ export const ChatFinishedConnector = ({ ref, ...props }) => (
 					color: customColor || color,
 					fontColor: customFontColor,
 					iconColor: customIconColor,
+					title: customTitle,
 				}}
-				title={I18n.t('Chat Finished')}
+				title={customTitle || I18n.t('Chat Finished')}
```

Comment:
Let's keep the other containers(other than Register, Chat and LeaveAMessage containers) with their own titles, okay?
Let's skip this one.